### PR TITLE
Updated code-of-conduct-bn with native translation. (bangla review requested)

### PR DESCRIPTION
**Pull request by jahir-raihan**
## What does this PR do?
It updates Code-Of-Conduct-bn version with native accent.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.

### CODE REVIEW

**Comment by eshellman** [comment]
Bangla review requested

**Comment by jahir-raihan** [comment]
@eshellman Hello, I'm from Bangladesh as well.

**Comment by eshellman** [comment]
yes, would like to have another bangla native to review the changes

**Comment by jahir-raihan** [comment]
Ok, but If there's no one to help within 6 hours, can I bring someone from my connection to review it?

**Comment by souravdasslg** [comment]
I can review :) I'm a native Bengali, from Kolkata.

**Comment by jahir-raihan** [comment]
Sure, Thank you.

File: docs/CODE_OF_CONDUCT-bn.md [hunk 1]
```diff
@@ -1,25 +1,26 @@
-এই প্রকল্পের অবদানকারী এবং রক্ষণাবেক্ষণকারী হিসাবে, এবং একটি উন্মুক্ত এবং স্বাগত সম্প্রদায়কে উত্সাহিত করার স্বার্থে, আমরা প্রতিশ্রুতি দিচ্ছি যে সমস্ত লোকেদের যারা রিপোর্টিং সমস্যা, বৈশিষ্ট্য অনুরোধ পোস্ট করা, ডকুমেন্টেশন আপডেট করা, পুল অনুরোধ বা প্যাচ জমা দেওয়া এবং অন্যান্য কার্যকলাপের মাধ্যমে অবদান রাখে।
+এই প্রকল্পের অবদানকারী এবং রক্ষণাবেক্ষণকারী হিসাবে, এবং একটি উন্মুক্ত এবং স্বাগত সম্প্রদায়কে উৎসাহিত করার স্বার্থে, আমরা সম্মান জানাই  সে  সমস্ত ব্যক্তিদের যারা সমস্যা রিপোর্টিং, নতুন বৈশিষ্ট যোগের আবেদন , ডকুমেন্টেশন আপডেট , নতুন বৈশিষ্ট্য যোগ করে বা প্যাচ জমা দেয় এবং অন্যান্য কার্যকলাপের মাধ্যমে অবদান রাখে।
```
Comment (souravdasslg) [inline docs/CODE_OF_CONDUCT-bn.md:1]: Let's remove `এবং স্বাগত`. Doesn't make sense here.

Comment (souravdasslg) [reply docs/CODE_OF_CONDUCT-bn.md:1]: `নতুন বৈশিষ্ট্য যোগ করে বা প্যাচ জমা দেয়` --> `নতুন বৈশিষ্ট্য যোগ করে বা প্যাচ জমা দেওয়া ` sounds more good.

File: docs/CODE_OF_CONDUCT-bn.md [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
 
+- যৌনতামূলক ভাষা বা চিত্রের ব্যবহার
+- ব্যক্তিগত আক্রমণ
+- ট্রোলিং বা অপমানজনক মন্তব্য
+- জনসম্মুখে  বা ব্যক্তিগত হয়রানি
+- স্পষ্ট অনুমতি ছাড়াই অন্যের ব্যক্তিগত তথ্য, যেমন শারীরিক বা ইন্টারনেট ঠিকানা প্রকাশ করা
+- অন্যান্য অনৈতিক বা পেশাগত আচরণ
 
-প্রজেক্ট রক্ষণাবেক্ষণকারীদের এই আচরণবিধির সাথে সংযুক্ত নয় এমন মন্তব্য, প্রতিশ্রুতি, কোড, উইকি সম্পাদনা, সমস্যা এবং অন্যান্য অবদানগুলিকে অপসারণ, সম্পাদনা বা প্রত্যাখ্যান করার অধিকার এবং দায়িত্ব রয়েছে, বা অন্যান্য আচরণের জন্য অস্থায়ী বা স্থায়ীভাবে কোনো অবদানকারীকে নিষিদ্ধ করার অধিকার রয়েছে। তারা অনুপযুক্ত, হুমকি, আপত্তিকর, বা ক্ষতিকারক বলে মনে করে।
+প্রজেক্ট রক্ষণাবেক্ষণকারীদের এই আচরণবিধির সাথে সংযুক্ত নয় এমন মন্তব্য, প্রতিশ্রুতি, কোড, উইকি সম্পাদনা, সমস্যা এবং অন্যান্য অবদানগুলিকে অপসারণ, সম্পাদনা বা প্রত্যাখ্যান করার অধিকার এবং দায়িত্ব রয়েছে, বা অন্যান্য আচরণের জন্য অস্থায়ী বা স্থায়ীভাবে কোনো অবদানকারীকে নিষিদ্ধ করার অধিকার রয়েছে যা তারা অনুপযুক্ত, হুমকি, আপত্তিকর, বা ক্ষতিকারক বলে মনে করে।
```
Comment (souravdasslg) [inline docs/CODE_OF_CONDUCT-bn.md:14]: remove `তারা` in last line.

File: docs/CODE_OF_CONDUCT-bn.md [hunk 3]
```diff
@@ -1,25 +1,26 @@
...
 এই আচরণবিধি গ্রহণ করার মাধ্যমে, প্রকল্পের রক্ষণাবেক্ষণকারীরা এই প্রকল্প পরিচালনার প্রতিটি ক্ষেত্রে এই নীতিগুলিকে ন্যায্যভাবে এবং ধারাবাহিকভাবে প্রয়োগ করার জন্য নিজেদের প্রতিশ্রুতিবদ্ধ করে। প্রজেক্ট রক্ষণাবেক্ষণকারী যারা আচরণবিধি অনুসরণ করে না বা প্রয়োগ করে না তাদের প্রকল্প দল থেকে স্থায়ীভাবে সরিয়ে দেওয়া হতে পারে।
 
 এই আচরণবিধি প্রজেক্ট স্পেস এবং পাবলিক স্পেস উভয় ক্ষেত্রেই প্রযোজ্য হয় যখন একজন ব্যক্তি প্রকল্প বা তার সম্প্রদায়ের প্রতিনিধিত্ব করেন।
 
-gmail.com-এ victorfelder-এ একজন প্রজেক্ট রক্ষণাবেক্ষণকারীর সাথে যোগাযোগ করে আপত্তিজনক, হয়রানি, বা অন্যথায় অগ্রহণযোগ্য আচরণের ঘটনাগুলি রিপোর্ট করা যেতে পারে। সমস্ত অভিযোগ পর্যালোচনা করা হবে এবং তদন্ত করা হবে এবং পরিস্থিতির জন্য প্রয়োজনীয় এবং উপযুক্ত বলে মনে করা হবে। রক্ষণাবেক্ষণকারীরা একটি ঘটনার প্রতিবেদকের বিষয়ে গোপনীয়তা বজায় রাখতে বাধ্য।
 
-এই কোড অফ কন্ডাক্ট কন্ট্রিবিউটর কভেন্যান্ট, সংস্করণ 1.3.0 থেকে অভিযোজিত হয়েছে, এখানে উপলব্ধ।
- https://contributor-covenant.org/version/1/3/0/
-Translations
+কেও যদি আপত্তিজনক, হয়রানি বা অগ্রহণযোগ্য আচরণের শিকার হয়, তাহলে যে কোন একজন প্রজেক্ট রক্ষণাবেক্ষণকারীর সাথে `victorfelder[@]gmail.com` যোগাযোগ করার অনুরোধ করা হল। যোগাযোগ কারীর তথ্য অবশ্যই গোপন রাখা হবে।
```
Comment (souravdasslg) [inline docs/CODE_OF_CONDUCT-bn.md:21]: remove যে কোন একজন, because the original document mentions the particular owner. Also, that is grammatically wrong.

**Review by souravdasslg — CHANGES_REQUESTED** [review]
Some minor changes in the doc to make it more natural.

File: docs/CODE_OF_CONDUCT-bn.md (hunk 3 above)
Comment (jahir-raihan) [reply docs/CODE_OF_CONDUCT-bn.md:21]: ok

Comment (jahir-raihan) [reply docs/CODE_OF_CONDUCT-bn.md]: @souravdasslg Give it a review again please.

**Review by souravdasslg — APPROVED** [review]
LGTM

**Comment by jahir-raihan** [comment]
@eshellman Hello, It's reviewed now, please take a look.

**Comment by eshellman** [comment]
could you resolve the remaining suggestions? thanks everyone!

**Comment by jahir-raihan** [comment]
Everything was taken care of.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books, all who helped!

**Pull request merged**